src/clientsettingsdlg.cpp

```diff
@@ -169,12 +169,12 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient*         pNCliP,
     QString strSndCrdBufDelayTT = tr ( "If the buffer delay settings are "
         "disabled, it is prohibited by the audio driver to modify this "
         "setting from within the software. "
-        "On Windows, press the ASIO Setup button to open the "
+        "On Windows, press the ASIO Device Setup button to open the "
```

Comment:
FYI I renamed the button name to _ASIO Device **settings**_.

Reply:
Ah. Sorry. I will revert and rename it to the correct name

Reply:
The tooltips seemed to mention settings more often, so I just kept it consistent

Reply:
Ok. Can you please look at the new changes?

Reply:
looks good

Reply:
Great. Thanks. I will resolve this now.